src/main/java/mesosphere/marathon/client/model/v2/UpgradeStrategy.java

```diff
@@ -0,0 +1,31 @@
+package mesosphere.marathon.client.model.v2;
+
+
+import mesosphere.marathon.client.utils.ModelUtils;
+
+public class UpgradeStrategy {
+
+    private double minimumHealthCapacity = 0.5;
```

Comment:
we can reset them to 0, according to the documentation https://mesosphere.github.io/marathon/docs/deployments.html it would kill old instance before start new one. I believe we would be on the safe side with zeros, but i don't mind if i add two arg constructor as well. 
@mohitsoni wdyt?